Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node
+
+RUN npm install -g phantomjs
```

Comment:
It's a very recommanded practice to group RUN commands, so:
```Dockerfile
RUN npm install -g phantomjs \
 && npm install the-a11y-machine
```

Also, why not doing a global install (`-g`) of the-a11y-machine too?

would be more appropriate.

Also, NPM is pretty verbose by default... verbosity can be reduced with either conf file or better in a docker build, env vars, something like that for example:
```Dockerfile
ENV NPM_CONFIG_LOGLEVEL=warn \
    NPM_CONFIG_PROGRESS=false \
    NPM_CONFIG_SPIN=false
```